features/call/impl/src/main/kotlin/io/element/android/features/call/impl/utils/ActiveCallManager.kt

```diff
@@ -62,23 +60,22 @@ interface ActiveCallManager {
 
     /**
      * Hangs up the active call and removes any associated UI.
+     * @param callType The type of call that the user hung up, either an external url one or a room one.
      */
-    fun hungUpCall()
+    fun hungUpCall(callType: CallType)
```

Comment:
there seems to be a conflict between the docstring and the method name.
`hangUpCall` or
```
 * Called when the active call has been hung up and all associated UI was removed.
```